kubernetes/create-services.sh

```diff
@@ -2,10 +2,19 @@
 
 ./create-config-maps.sh
 
+kubectl create --record --filename services/verifier.yml
+./check-pods.sh "verifier" "true"
 kubectl create --record --filename services/geth.yml
+./check-pods.sh "geth"
 kubectl create --record --filename services/rabbitmq.yml
+./check-pods.sh "rabbitmq"
```

Comment:
We don't need to wait after each `create` separately. We're unnecessarily delaying the deployment. Wait only when you need to. You can move all the other waits after the `create`s.

Might be convenient to make it possible to pass multiple pod names to the `./check-pods.sh` script.

Reply:
Are you sure ? This script was created to wait for the creation of a statefulset pods before creating the deployment pods. This behavior allows to the best distribution of resources. When we will implement your version, all deployment pods will be create before the statefulset pods, so there will be no resources for statefulset pods

Reply:
Done.

Reply:
That's why I said "wait only when you need to". Create stateful sets, wait for them, then create everything else, wait for it. No need to wait after every pod separately when others could be created at the same time.